### PR TITLE
Button

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
made  icon button and floating action button

### CODE REVIEW

**Review by Shubhamkr2610 — APPROVED** [review]
appreciation: good naming with classes . Code looks good and readable . Easy to understand , you can merge it

**Review by ApurvaSawant11 — APPROVED** [review]
The code looks clean and readable. Files are also properly indented. Good work!

File: components/badge.css [hunk 1]
```diff
@@ -11,6 +11,23 @@
     text-align: center;
     border-radius: 0.3rem;
```
Comment (bossmonkey98) [inline components/badge.css:12]: Could use px here instead of rem

File: components/badge.css [hunk 2]
```diff
@@ -11,6 +11,23 @@
...
 }
+.badge-status{
+    padding: 0.5rem;
```
Comment (bossmonkey98) [inline components/badge.css:15]: Same here

**Review by bossmonkey98 — APPROVED** [review]

**Pull request merged**